### DISCUSSION TITLE
Images carousel slideshow

### DISCUSSION THREAD

**Discussion by jitesha653** [EvotecIT/PSWriteHTML#378] [General]
Is there any way how to we add a set images  in a folder of in slideshow  with carousel slideshow

**Comment by PrzemyslawKlys** [comment]
New-HTMLCarousel and New-CareouselSlide could work, along with New-HTMLImage but I've never tried it. It's on on my todo list to have example for it, and if that doesn't work automatically to provide something like carousel and other slideshows.

**Reply by jitesha653** [reply]
I have tried this but its not working2, but i am able to get images in New-HTMLAccordion and New-AccordionItem
tried the same logic with Carousel, but didnt work
